tests/model/test_sam_resources.py

```diff
@@ -691,3 +693,46 @@ def test_unsupported_permissions_combination(self):
             "Unsupported 'Permissions' provided for connector from AWS::SQS::Queue to AWS::Lambda::Function; valid combinations are: Read \\+ Write.",
         ):
             connector.to_cloudformation(**self.kwargs)[0]
+
+
+class TestGraphQLApiNoneDataSource(TestCase):
```

Comment:
Unless there's any benefit to these tests, I'd much rather have all these tests as regular transform tests so they go through the schema tests and the full transform (there could be side effects in other parts of transform that these tests would not catch).

Reply:
These are also in transform tests `graphqlapi_multiple_none_datasource_functions.yaml`